rosdep/python.yaml

```
@@ -9113,6 +9114,7 @@ python3-transforms3d:
   debian:
     pip:
       packages: [transforms3d]
+  nixos: [python3Packages.transforms3d]
   fedora: [python3-transforms3d]
```

This needs to be in alphabetical order:

```suggestion
  fedora: [python3-transforms3d]
  nixos: [python3Packages.transforms3d]
```